### PR TITLE
Add "mkdocs" generation for Kompose site

### PR DESCRIPTION
**Pull request by cdrage**
This commit adds "mkdocs" generation to push markdown documenation to
the Kompose website.

On each push to master, new documentation is automatically generated and
pushed to "gh-pages"

### CODE REVIEW

**Comment by cdrage** [comment]
**NOTE** Travis will fail.

Before this is merged and tested upon, this guide needs to be followed:

https://blog.wyrihaximus.net/2015/09/github-auth-token-on-travis/

GITHUB_API_KEY needs to be added as an environment variable to the travis builds.

**Comment by cdrage** [comment]
This will create https://skippbox.github.io/kompose with automatic doc generation on each master PR merge.

**Comment by sebgoa** [comment]
@cdrage thanks, I will take this one.

Can you sign a CLA from https://github.com/skippbox/cla

Maybe check with @pradeepto or @dustymabe , I just want to keep track of CLAs so if we need to change the copyright it will be easy.

**Comment by cdrage** [comment]
@sebgoa ya sure, so what do i need to do? fill out out and send it to some email?

**Comment by sebgoa** [comment]
@cdrage yes just sign, put your name and address, scan and send to: info at skippbox dot com

**Comment by cdrage** [comment]
@sebgoa sent

File: docs/README.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
-# Kompose Documentation
-
-
-* [User Guide](user-guide.md)
-* [Architecture](architecture.md)
-* [Development](development.md)
+# Docs
 
+This is automatically created my `mkdocs`! Please checkout [https://skippbox.github.io/kompose](https://cdrage.github.io/kompose) for documentation on Kompose.
```
Comment (sebgoa) [inline docs/README.md]: created "by"

File: script/pushdocs.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
```
Comment (sebgoa) [inline script/pushdocs.sh]: ideally we need to push a license header here.

**Comment by cdrage** [comment]
@sebgoa updated with your changes!

Remember to login to Travis and add the API key via the guide before mergin'

**Comment by sebgoa** [comment]
@cdrage I did not forget about this. But now that we moved to the incubator I am trying to do a few things first.

In any case, while you signed the skippbox CLA, now that we moved you need to sign the CNCF CLA.

maybe @dustymabe can help you with that.

**Comment by dustymabe** [comment]
@cdrage it's pretty easy - just follow what I said here: https://github.com/kubernetes-incubator/kompose/pull/211#issuecomment-254308960

**Comment by cdrage** [comment]
Since we've moved to the kubernetes-incubator, I'm going to close this *for now* until I figure out a good area to push the docs to (the kubernetes site, or some other area)

**Pull request closed without merging**